pkg/geo/geomfn/topology_operations.go

```diff
@@ -188,3 +193,75 @@ func SharedPaths(a geo.Geometry, b geo.Geometry) (geo.Geometry, error) {
 	}
 	return gm, nil
 }
+
+// Node Returns a geometry containing a set of linestrings using the least possible number of nodes while preserving all of the input ones.
```

Comment:
nit: let's break this up:

```
// Node returns a geometry containing a set of LineStrings using the least
// possible number of nodes while preserving all of the input ones.

```

Also, feel free to move this function and it's tests to node.go.

Reply:
that's a good idea 👍